modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -190,7 +199,8 @@ public AssignmentsImpl(Map<String, AssignmentIngestionExecutor> executors,
         this.assignmentsRepository = assignmentsRepository;
         this.mutexRepository = mutexRepository;
         this.eventLog = eventLog;
-  
+        this.metadataCacheEnabled=metadataCacheEnabled;
```

Comment:
q: Did you use either Eclipse/Intellij to format for all new code? It will automatically avoid this kind of comment from @iizrailevsky. Also, it will help another PR that @AndreaSuckro was working on to format all codes using either Eclipse or Intellij.